### PR TITLE
CSS Fixes

### PR DESCRIPTION
**Pull request by rowsky**
# CSS Fixes

This PR aim to fix some CSS problems. Abnormal display of contents, non responsiveness ...

First commit: Fixes this bug with long autocompletion lists of users :
image

### CODE REVIEW

**Comment by automation-wavekeeper** [comment]
### New commit for responsive logo:
- The image is replaced by a background that covers a span, if the screen width is below 768, the "Borgia" part of the logo is hidden.

image

**Comment by rowsky** [comment]
### New commit for responsive logo:
- The image is replaced by a background that covers a span, if the screen width is below 768, the "Borgia" part of the logo is hidden.

image

**Comment by automation-wavekeeper** [comment]
TEST (spam)

File: templates/base_clean.html [hunk 1]
```diff
@@ -36,15 +36,14 @@
     <div class="container-fluid">
       <div class="navbar-header">
         <a class="navbar-brand" href="{% if user.is_authenticated %}{% url 'url_group_workboard' group_name='gadzarts' %}{% endif %}">
-           <span class="logo"></span>
```
Comment (AlexandrePalo) [inline templates/base_clean.html]: Hmm ?

**Comment by rowsky** [comment]
### Fixing login form position
Went from this
image
To this
image

File: templates/base_clean.html (hunk 1 above)
Comment (rowsky) [reply templates/base_clean.html]: The logo on the navbar is not used in base clean template

**Comment by rowsky** [comment]
### Fixing navbar
- Collapsible navbar with minimum html changes

image

**Comment by rowsky** [comment]
### New lateral menu
- The previous version of the lateral menu used to work fine for desktop, but it was difficult to handle for mobile.
- The new menu is an offcanvas multi-level push menu, it pushes all the content (except footer) when it is active.
- The menu colors are compatible with less variables.
- There is still a thing to solve for mobile which is the fact that there are now two menus. The two must merge somehow ...

image

**Comment by rowsky** [comment]
Fixes #55 
Ready for merge.
Other css fixes will handle other issues.

**Pull request merged**